main.go

```diff
@@ -94,7 +98,7 @@ func PrepareCmd() *cobra.Command {
 
 			err = yaml.Unmarshal(customizationFileBytes, &customization)
 			if err != nil {
-				logger.Error().Err(err).Msg("failure to parse customization file")
+				logger.Error().Err(err).Msg(FailureToParseCusromizationFile)
```

Comment:
L:91 ->logger.Error().Err(err).Msg(FailureToReadConfigurationFile)

Can we replace lines 91-103 and 39-49 by a helper function ?